packages/frontend/src/main/components/comments/CommentEditor.vue

```diff
@@ -49,8 +48,11 @@ import {
 import FileUploadProgress from '@/main/components/common/file-upload/FileUploadProgress.vue'
 import { UploadFileItem } from '@/main/lib/common/file-upload/fileUploadHelper'
 import { differenceBy } from 'lodash'
+import { useQuery } from '@vue/apollo-composable'
+import { serverInfoBlobSizeLimitQuery } from '@/graphql/server'
```

Comment:
You should import the query from '@/graphql/generated/graphql' instead, then it'll have the proper types and everything. Currently I'm not sure how the types are working for you on lines 90-95, I'd think that `result` just has type `any`